benchopt/utils/shell_cmd.py

```diff
@@ -3,7 +3,13 @@
 import subprocess
 
 from ..config import DEBUG
-from ..config import get_setting
+
+
+def get_setting(setting_name):
```

Comment:
@wassimmazouz it seems risky to have two functions with the same name; I don't see why we do it this way ?